examples/rmrk/resource/wasm/Cargo.toml

```diff
@@ -9,10 +9,14 @@ rmrk-resource-app = { path = "../app" }
 [build-dependencies]
 gwasm-builder.workspace = true
 rmrk-resource-app = { path = "../app" }
+sails-client-gen.workspace = true
 sails-idl-gen.workspace = true
 
 [dev-dependencies]
 gtest.workspace = true
+parity-scale-codec = { workspace = true, features = ["derive"] }
```

Comment:
This shouldn't be needed. If it is, then there is some issue which needs be fixed. The same is about `scale-info`. I can guess the issue is in the `client-gen` which doesn't make use of `#[codec(crate = sails_rtl::scale_codec)]` and `#[scale_info(crate = sails_rtl::scale_info)]` for re-exported stuff